src/main/shop/Main.java

```diff
@@ -0,0 +1,35 @@
+import service.*;
+import strategy.AddOperationHandler;
+import strategy.BalanceOperationHandler;
+import strategy.OperationHandler;
+import strategy.PurchaseOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String pathToStartFile = "src/main/shop/resources/test.csv";
+    private static final String pathToResultFile = "src/main/shop/resources/result.csv";
```

Comment:
is it the correct naming for constants?
also, please place the resources in `src/main/resources` folder